### PR TITLE
feat: add initial terraform module config

### PR DESCRIPTION
**Pull request by Dennisvandermeulen**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  tags = merge(
+    try(var.tags),
+    tomap({
+      "Resource Type" = "Log Analytics Workspace"
+    })
+  )
+}
+
+module "storage_account" {
+  source = "github.com/schubergphilis/terraform-azure-mcaf-storage-account.git" #TODO: change source
```
Comment (Blankf) [inline main.tf:27]: should this still be todo or ?

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    tomap({
+      "Resource Type" = "Storage Account"
+    })
+  )
+}
+
+resource "azurerm_storage_management_policy" "this" {
+  count = var.storage_account != null ? 1 : 0
+
+  # TODO: Expand these rules with tiering settings
```
Comment (Blankf) [inline main.tf:47]: same, should there still be some work done?

**Review by Blankf — COMMENTED** [review]
looks good, but 2 comments

File: main.tf (hunk 1 above)
Comment (Dennisvandermeulen) [reply main.tf:27]: https://github.com/schubergphilis/terraform-azure-mcaf-monitoring/issues/3 added as an issue for now

File: main.tf (hunk 2 above)
Comment (Dennisvandermeulen) [reply main.tf:47]: https://github.com/schubergphilis/terraform-azure-mcaf-monitoring/issues/4

created an issue for this, we should have a discussion about the necessity

**Pull request merged**